tests/unit/fixtures/latedef-esnext.js

```diff
@@ -60,3 +60,8 @@ let dc = {
         return dc;
     }
 };
+
+export function de() {}
+export class df {}
+export var dg;
+export let dh;
```

Comment:
yes, even though let and const now share the same code, there is no reason to leave it off, sorry I missed that.